docs/BUILD.md

```diff
@@ -1,6 +1,56 @@
 # Building s2n-tls
 
-s2n-tls can be built as follows:
+To use s2n-tls, you must build the library from the source and then include it in your program.
+## Requirements
+
+s2n-tls supports and tests on **x86** and **arm** architectures.
+
+### System requirements
+
+* 20GB RAM available
```

Comment:
I don't think this is a requirement. Was disk space meant rather than RAM? Even so, I'm not sure 20 GB is needed. Some additional space is needed when building, but the last release was only ~43 MB.